src/main/java/org/takes/facets/fork/TkFork.java

```diff
@@ -63,11 +63,17 @@
  * @see org.takes.facets.fork.FkRegex
  * @see org.takes.facets.fork.FkParams
  */
-@ToString(of = "forks")
-@EqualsAndHashCode(of = "forks")
+@ToString(of = TkFork.LOMBOK)
+@EqualsAndHashCode(of = TkFork.LOMBOK + "")
 public final class TkFork implements Take {
 
     /**
+     * Constant for the Lombok.
+     * Lombok doesn't include transient fields by default.
+     */
+    static final String LOMBOK = "forks";
```

Comment:
@yegor256 I am not sure, what is the right design.
We need the information in the pre-compiler as Annotations are handled in compile time. We cannot pass objects to annotations in my understanding.